CSharp/BatchExplorer/Models/JobModel.cs

```diff
@@ -304,7 +321,7 @@ public override List<PropertyModel> PropertyModel
         /// <summary>
         /// Delete this job
         /// </summary>
-        public async System.Threading.Tasks.Task DeleteAsync()
+        public async System.Threading.Tasks.Task DeleteAsync(bool refresh = true)
```

Comment:
Defaulting `refresh` to true means that existing calls to DeleteAsync() (with no args) will change their behaviour to now do a refresh - is that intended/desired?